### DISCUSSION TITLE
how do i host this on github pages

### DISCUSSION THREAD

**Discussion by Alex-A137** [ruffle-rs/ruffle#9259] [General]

**Comment by n0samu** [comment]
Hi, take a look here: https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site
And see my example here: https://github.com/n0samu/n0samu.github.io/tree/main/ruffle-testing

**Comment by Alex-A137** [comment]
dp you know how i could use this to play papas pizzaria
\

**Comment by Bumpf** [comment]
Hey! I've created a Github Pages site using Ruffle, but holding a huge repository of swf files is a little annoying :^). I am trying to figure out how to create an Upload button where users can upload their own swf files and Ruffle will just play those (yes I know flashpoint yadda yadda i just want to do it okay), but because Pages are static I cant host user files. Is there some way to take user files as just a normal input that will go away when the tab is closed or refreshed? I really can't figure it out, and it doesn't help that I don't know Javascript.
